src/Public/LogRhythm/Case/Update-LrCasePlaybookProcedure.ps1

```diff
@@ -0,0 +1,357 @@
+        [System.String]   ->  Status
+    .OUTPUTS
+        System.Object representing the returned LogRhythm playbook procedures on the applicable case.
+
+    .EXAMPLE
+        PS C:\> Update-LrCasePlaybookProcedure -Credential $Token -CaseId "F47CF405-CAEC-44BB-9FDB-644C33D58F2A"
+
+    .NOTES
+        LogRhythm-API
+    .LINK
+        https://github.com/SmartResponse-Framework/SmartResponse.Framework        
+    #>
+
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false, Position = 0)]
+        [ValidateNotNull()]
+        [pscredential] $Credential = $SrfPreferences.LrDeployment.LrApiToken,
+
+        [Parameter(
+            Mandatory = $true,
+            ValueFromPipeline = $true,
+            Position = 1
+        )]
+        [ValidateNotNullOrEmpty()]
+        [object] $CaseId,
+
+        [Parameter(
+            Mandatory = $false,
+            Position = 2
+        )]
+        [ValidateNotNullOrEmpty()]
+        [string] $PlaybookId,
+
+        [Parameter(
+            Mandatory = $true,
+            Position = 3
+        )]
+        [ValidateNotNullOrEmpty()]
+        [string] $ProcedureId,
+
+        [Parameter(
+            Mandatory = $false,
+            Position = 4
+        )]
+        [ValidateNotNullOrEmpty()]
+        [string] $Assignee,
+
+        [Parameter(
+            Mandatory = $false,
+            Position = 5
+        )]
+        [ValidateNotNullOrEmpty()]
+        [string] $Notes,
+
+        [Parameter(
+            Mandatory = $false,
+            Position = 6
+        )]
+        [ValidateNotNullOrEmpty()]
+        [string] $DueDate,
+
+        [Parameter(
+            Mandatory = $false,
+            Position = 7
+        )]
+        [ValidateNotNullOrEmpty()]
+        [string] $Status
+    )
+
+    Begin {
+        $Me = $MyInvocation.MyCommand.Name
+        
+        $BaseUrl = $SrfPreferences.LRDeployment.CaseApiBaseUrl
+        $Token = $Credential.GetNetworkCredential().Password
+
+        # Enable self-signed certificates and Tls1.2
+        Enable-TrustAllCertsPolicy        
+    }
+
+
+    Process {
+        # Get Case Id
+        $IdInfo = Test-LrCaseIdFormat $CaseId
+        if (! $IdInfo.IsValid) {
+            throw [ArgumentException] "Parameter [CaseId] should be an RFC 4122 formatted string or an integer."
+        }
+
+        # Get Case Guid
+        $CaseGuid = (Get-LrCaseById -Id $CaseId).id
+
+        
+
+        # Populate list of Case Playbooks
+        $CasePlaybooks = Get-LrCasePlaybooks -Id $CaseId
+
+        # Validate or Retrieve Playbook Id
+        if ($PlaybookId) {
+            if ($CasePlaybooks -eq $null) {
+                throw [ArgumentException] "No Playbooks located on case: $CaseId."
+            } else {
+                # Validate Playbook Id
+                $PlaybookType = Test-LrPlaybookIdFormat -Id $PlaybookId
+                # Step through array of Playbooks assigned to case looking for match
+                if ($CasePlaybooks -is [array]) {
```

Comment:
No need to change this, but thought I'd mention it - you don't need to check if `$CasePlaybooks` is an array.  Just go right into a `foreach` loop.  If only one result was returned, the loop will only execute once.  

```powershell
# 2068 only has one playbook assigned
PS> $playbooks = Get-LrCasePlaybooks -Id 2068
PS> foreach($pb in $playbooks2) { $pb.id }
0ECC5442-919C-4D7C-8162-D0BCC689642B
```